pkg/aws/aws.go

```diff
@@ -251,29 +256,24 @@ func (c *Client) IamRoleTrust(action, entityName, roleName string) error {
 		return err
 	}
 
-	_, err = UpdateIAMAssumeRole(roleName, policyJSON, iamClient)
+	_, err = UpdateIAMAssumeRole(roleName, policyJSON, c.IAMClient)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (c *Client) EfsCsiRoleTrust(action, roleName string) error {
-	var trustedEntityName = fmt.Sprintf("efs-csi-role-%s", BDDClusterName)
-	return c.IamRoleTrust(action, trustedEntityName, roleName)
-}
```

Comment:
I think this is too specific to have it here. I agree that is not a good idea to have the full role name in the `.feature` file like you mentioned. Lets implement this function directly in arktika, we use kubedog's step from it.

Reply:
Ok, sounds good tome